cli/pom.xml

```diff
@@ -45,7 +45,6 @@
             <artifactId>iotdb-server</artifactId>
             <version>${project.version}</version>
             <type>test-jar</type>
-            <scope>test</scope>
```

Comment:
The client cannot directly import iotdb-server and jar. need deal...

Reply:
Using shade plugin shaded DatetimeUtils.class into iotdb-client.jar